edgelessdb/getting-started/quickstart-azure.md

```diff
@@ -0,0 +1,91 @@
+# Quickstart: Microsoft Azure
+In this guide, you will set up EdgelessDB using the Microsoft Azure Marketplace.
+
+## Get the machine
+Visit the Azure Marketplace and get [EdglessDB](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/edgelesssystems.edb?tab=Overview).
+
+![azure_create](../_assets/azure/azure_get.png)
```

Comment:
Seems like docsify ignores relativePath config for images, so path must be `(_media/azure_get.png)`. Unfortunately, the preview in vscode then no longer works. So sad.

Reply:
* Pls use reasonable alt texts for the images
* Rename azure_wizard_edited.png to azure_wizard.png
* azure_create_edited.png is unused

Reply:
> Seems like docsify ignores relativePath config for images

We are using a [custom render function for images](https://github.com/edgelesssys/docs.edgeless.systems/blob/b27e20a29a767cc9789e3bf820b12e5f431e59fa/edgelessdb/index.html#L58), but I guess it should work for `relativePath`, since the `relativePath` option was deleted from the MarbleRun `index.html` at a later point. 
I can't remember why we deleted this option for MarbleRun, but we should try to restore consistency between those index files to prevent such mistakes.